### ISSUE TITLE
[Bug] Alert after each "Update imports"

### ISSUE THREAD

**Issue by max-mykhailenko** [unlight/sublime-import-helper#17]
```
Import Helper
{ Error: ENOENT: no such file or directory, stat '/Users/max/Work/Projects/pet-project/storage/orders/189/success/31/files/01.jpg'
  errno: -2,
  code: 'ENOENT',
  syscall: 'stat',
  path: '/Users/max/Work/Projects/pet-project/storage/orders/189/success/31/files/01.jpg' }
```

**Comment by unlight** [comment]
Is that all?
It supposed to be a stack trace.

**Comment by max-mykhailenko** [comment]
Yes, in console I also see only that message

**Comment by unlight** [comment]
@max-mykhailenko
Apparently this is some kind of environment specific issue. 
Can you provide more info about your folder structure?
* Content of the file where `...files/01.jpg` was found
* Content of `*.sublime-project` file

**Comment by max-mykhailenko** [comment]
Hah, it's a link to the not existing file.

**Comment by unlight** [comment]
That's make sense, this case should be handled.

**Comment by unlight** [comment]
Should be fixed in v1.3.0, just released
I'm on Windows, so I'm not able to test file symlink, but directory symlink error handled fine.
Please, give some feedback if error still happens.

**Comment by max-mykhailenko** [comment]
Works without errors. Thanks!
